src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMFSC

```diff
@@ -9,21 +9,31 @@
   },
   "items": [
     {
+      "item": 1,
       "name": "REF_SURF_CONC",
       "value_type": "DOUBLE",
       "dimension": "FoamSurfactantConcentration"
     },
     {
+      "item": 2,
       "name": "EXPONENT",
       "value_type": "DOUBLE",
       "default": 1,
       "dimension": "1"
     },
     {
+      "item": 3,
       "name": "MIN_SURF_CONC",
       "value_type": "DOUBLE",
       "default": 1e-20,
       "dimension": "FoamSurfactantConcentration"
+    };
+      "item": 4,
```

Comment:
I guess this should be
```
},
{
  "item": 4,
```
instead.  May I suggest you use an online JSON syntax checker such as https://jsonlint.com/ when doing this work to avoid these kinds of issues?

Reply:
Use of e.g. the suggested syntax checker seems like a good idea when debugging json syntax problems, but the first step is just to build locally - that will catch json syntax errors, although it gives no clue to where the problem is.